### PR TITLE
Add pretty pagination

### PR DESCRIPTION
**Pull request by B1acksheep**
This commit will replace the "Click and Insert Number" pagination with an extended version.

### CODE REVIEW

**Comment by alphartdev** [comment]
Hi, I'm sorry I'm really running out of time right now but I still need to review the code. Again I really apologize for taking so long.

File: application/views/_template/header.php [hunk 1]
```diff
@@ -43,7 +43,6 @@
 			</div>
 		</div>
 	</nav>
-<?php include("application/views/_template/modal-info.php");?>
-<div class="row">
-<div class="col-md-1"></div>
```
Comment (alphartdev) [inline application/views/_template/header.php]: Just a bit of doubt about remove the col, it may be responsible for alignement?

**Comment by alphartdev** [comment]
Just tested it, it's far better than the original one. Just concerning the col div removal, did you notic any issues about this or is this fine on your side (it's fine on mine, just want to have more inputs)?

**Comment by B1acksheep** [comment]
We have no problems with it. Everything is nice and tidy. :)

**Comment by alphartdev** [comment]
Great then let's merge that :D

**Comment by alphartdev** [comment]
Great job and thanks for the PR!

**Comment by B1acksheep** [comment]
As long as it helps the plugin. Gladly again!

**Pull request merged**